WordPress/src/main/java/org/wordpress/android/models/Post.java

```diff
@@ -395,12 +395,12 @@ public String getQuickPostType() {
      * the user first publishes a post
      * @return
      */
-    public boolean hasChangedFromLocalDraftToPublished() {
-        return mChangedFromLocalDraftToPublished;
+    public boolean hasChangedFromDraftToPublished() {
```

Comment:
What was the reason for renaming this? I think `hasChangedFromLocalDraftToPublished` was a more accurate description.